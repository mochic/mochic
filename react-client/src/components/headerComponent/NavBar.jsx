import React from 'react';
import { Link, Route } from 'react-router-dom';

const NavBar = () => (
  <header>
    <ul id="headerButtons">
      <li className="navButton">
        <Link to="/">home</Link>
      </li>
    </ul>
  </header>
);

export default NavBar;
