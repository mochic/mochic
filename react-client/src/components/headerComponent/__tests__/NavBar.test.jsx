import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import NavBar from '../NavBar';
import renderer from 'react-test-renderer';

it('renders deep', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <NavBar />
    </MemoryRouter>
  );
  expect(tree).toMatchSnapshot();
});
