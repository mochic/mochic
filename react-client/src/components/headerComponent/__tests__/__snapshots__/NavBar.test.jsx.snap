// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders deep 1`] = `
<header>
  <ul
    id="headerButtons"
  >
    <li
      className="navButton"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        home
      </a>
    </li>
  </ul>
</header>
`;
